src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -5433,4 +5433,7 @@ Please consider migrating '{0}' to `PackageReference` and using the pack targets
 You can set the '{1}' environment variable to 'true' to temporarily reenable this functionality.</value>
     <comment>Please do not localize `project.json` pack and `PackageReference`. 0 - path, 1 - env var name</comment>
   </data>
+  <data name="Error_NuGetExeNeedsToBeUnblockedAfterDownloading" xml:space="preserve">
+    <value>NuGet.exe file on path {0} needs to be unblocked after downloading : {1}</value>
```

Comment:
@JonDouglas 
Not sure if we need to include path since `error message already include path, we just need one, but couldn't decide which one to keep.

Reply:
The path should be the executing .exe path. I can't honestly tell from this screenshot. They look the same just that the `C:\` path is the more empowering one for the user.

Reply:
Extra info is: 
![image](https://user-images.githubusercontent.com/8766776/154165296-40b38cf7-de29-4068-ab95-a95411126867.png)
Not sure if users care about it.

Reply:
I removed extra part now it looks like below:
<img width="851" alt="image" src="https://user-images.githubusercontent.com/8766776/154166660-3aaa5172-5eb6-4579-aed5-d9aa9837b402.png">